packages/components/autocomplete/src/autocomplete.vue

```diff
@@ -29,6 +29,8 @@
       <el-input
         ref="inputRef"
         v-bind="attrs"
+        :clearable="clearable"
+        :disabled="disabled"
```

Comment:
Perhaps you should also add the `name` attribute

Reply:
Nice catch~